### ISSUE TITLE
Migrate Facts and ENC to Puppet guide

### ISSUE THREAD

**Issue by maximiliankolb** [theforeman/foreman-documentation#579]
Migrate content from the Foreman Manual (https://theforeman.org/manuals/2.5/index.html#3.5.5FactsandtheENC) to the Foreman Documentation (https://docs.theforeman.org/nightly/Managing_Hosts/index-foreman-el.html#using-puppet-for-configuration-management_managing-hosts).

**Comment by melcorr** [comment]
Thanks @maximiliankolb !

**Comment by melcorr** [comment]
If someone could provide me with a screen of this UI, I'll rework the manual docs into a module for the new Puppet guide here.

**Comment by lzap** [comment]
@ares I think you mentioned some always-on Foreman instance, can Mel access it? I could sync some Puppet modules for her I remember you use Salt/Chef

**Comment by ares** [comment]
Of course, I'll share the access offline. There are some puppet artifacts already. There's no Salt/Chef though ATM.

**Comment by maximiliankolb** [comment]
@maximiliankolb Recheck if this is still missing.

**Comment by maximiliankolb** [comment]
Foreman manual: https://theforeman.org/manuals/3.10/index.html#3.5.5FactsandtheENC
